cmd/influxd/run/server_test.go

```diff
@@ -1948,6 +1948,219 @@ cpu value=20 1278010021000000000
 	}
 }
 
+// Ensure the server can handle various group by time difference queries.
+func TestServer_Query_SelectGroupByTimeDifference(t *testing.T) {
+	t.Parallel()
+	s := OpenServer(NewConfig())
+	defer s.Close()
+
+	if err := s.CreateDatabaseAndRetentionPolicy("db0", newRetentionPolicyInfo("rp0", 1, 1*time.Hour)); err != nil {
```

Comment:
I just copied the setup function for `SelectGroupByTimeDerivative` which uses this for some reason.